lms/static/js/student_profile/views/learner_profile_view.js

```diff
@@ -9,7 +9,8 @@
 
         var LearnerProfileView = Backbone.View.extend({
 
-            initialize: function () {
+            initialize: function (options) {
+                this.options = _.extend({}, options);
```

Comment:
Due to the change in how Backbone handles options, a lot of views needed to be changed by adding the following code in initialize method. It would be better if we could move this logic to the base view. This will remove a lot of individual changes and also make views future proof.